eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -147,11 +150,10 @@ jobs:
           - Windows.11.Amd64.Client.Open
         # libraries on coreclr (outerloop and innerloop), or libraries on mono innerloop
         - ${{ if or(ne(parameters.jobParameters.testScope, 'outerloop'), ne(parameters.jobParameters.runtimeFlavor, 'mono')) }}:
+          - Windows.Amd64.Server2022.Open
+          - Windows.Server2025.Amd64.Open
```

Comment:
I would just replace 2022 with 2025 here. For outerloop it's debatable that we should test both, but I don't think running both on every PR makes sense.

Reply:
That makes sense.

Reply:
Shall I remove x64 too? (I'll just go ahead and do that)